lib/posix/Kconfig

```diff
@@ -143,6 +143,7 @@ config APP_LINK_WITH_POSIX_SUBSYS
 config EVENTFD
 	bool "Support for eventfd"
 	depends on !ARCH_POSIX
+	select POLL
```

Comment:
I wonder if this is appropriate use of `imply` instead of `select`.

Reply:
yeah, never sure when `imply` is the right option vs `select`.  Seems like its either `depends` or `select` as the eventfd code needs `POLL` to function.

Reply:
Probably `select` in this case then.